### DISCUSSION TITLE
git clone in moosefs mount is very slow

### DISCUSSION THREAD

**Discussion by sevenpy** [moosefs/moosefs#324] [Q&A]
hello,  I am new moosefs user, I have one question as below:
git clone in moosefs mount path is very slow, receiving objects only 1 MiB/s, but git clone use local disk receiving objects have 20+ MiB/s

this is git issue or moosefs fuse mount not support git useage?
my environment have one master server and three chunk server

thanks！

**Comment by chogata** [comment]
Git works on a large number of small files and performs a lot of operations on those small files. Such type of i/o will always work faster on a local hard drive than on any network file system, MooseFS included.
